salt/syspaths.py

```diff
@@ -102,15 +102,15 @@ def _get_windows_root_dir():
     if root_dir["success"]:
         # Make sure vdata contains something
         if root_dir["vdata"]:
-            return root_dir["vdata"]
+            return os.path.expandvars(root_dir["vdata"])
 
     # If this key does not exist, then salt was not installed using the
     # new method installer. Could be pip or setup.py or an older version of the
     # installer.
     log.debug("Failed to get ROOT_DIR from registry. %s", root_dir["comment"])
     # Check for C:\salt\conf
     old_root = "\\".join([os.environ["SystemDrive"], "salt", "conf"])
-    dflt_root = os.path.join(os.environ["ProgramData"], "Salt Project", "salt")
+    dflt_root = os.path.join(os.environ["ProgramData"], "Salt Project", "Salt")
```

Comment:
Since you are capitalizing Salt here, does the below also need salt capitalized?:

```
    root_dir = salt.utils.win_reg.read_value(
        hive="HKLM", key="SOFTWARE\\Salt Project\\salt", vname="root_dir"
    )
```

Reply:
The installer puts it in Capitalized, but since Windows is NOT case-sensitive... it doesn't really matter. I was just trying to keep them consistent